src/app/grid/grid-paste/grid-paste-sample.component.scss

```diff
@@ -25,14 +13,6 @@
     float: right;
 }
 
-.header-icon {
-    font-size: 1.4em;
-    width: 1.1em;
-    height: 1.1em;
-    float: right;
-    cursor: pointer;
-}
-
 .header {
```

Comment:
there's no `header` class applied anywhere in the template, so this likely doesn't do anything. Same for the `igx-grid__th` styles I think